src/App.js

```diff
@@ -42,7 +44,19 @@ import {getServerUrl, Legalese, serverGet} from './OctReactUtils';
 
 const qs = require('qs');
 
+const LEGACY_PATHS = [
+  '/',
+  '/confirmed',
+  '/detected',
+  '/labelImage',
+  '/preferences',
+  '/prototypes',
+  '/selected',
+  '/wildfirecheck'
+]
+
 function FirePagesHeader(props) {
+  console.log(props)
```

Comment:
Debug comment that can be deleted or is it useful to leave around?

Reply:
Yeah, usually I have my linter error out on `console.*` calls :sigh:

Reply:
Done